hibike/hibike_simulator.py

```diff
@@ -1,133 +1,144 @@
 import time, random
```

Comment:
I'm assuming Yizhe's version of this file is authoritative, so not reviewing anything here.